lib/iex/resources/resource.rb

```diff
@@ -14,6 +14,18 @@ def self.float_to_percentage(float_number)
         ].join
       end
 
+      # Useful for values that are already a percent but we want to convert into a 2 decimal place string
+      def self.percentage_to_string(float_percent)
```

Comment:
This is essentially a direct copy of `float_to_percentage` but the the values being returned from IEX for historical prices were already a percent just in float format, so the string representation ended up being an incorrect value. Happy to try and combine these two methods with an optional parameter or something different? Don't love the copy paste nature of this.

Reply:
I think copy paste is fine I wouldn't worry about it.